pymobiledevice3/services/mobile_config.py

```diff
@@ -23,6 +27,14 @@ def hello(self) -> None:
     def flush(self) -> None:
         self._send_recv({'RequestType': 'Flush'})
 
+    def escalate(self, pkcs12Bytes: bytes, password: str) -> None:
+        p12 = load_pkcs12(pkcs12Bytes, password.encode('utf-8'))
+
+        escalateResult = self._send_recv({"RequestType": "Escalate", "SupervisorCertificate": p12.cert.certificate.public_bytes(Encoding.DER)})
+        signedChallenge = PKCS7SignatureBuilder().set_data(escalateResult.get('Challenge')).add_signer(p12.cert.certificate, p12.key, hashes.SHA256()).sign(Encoding.DER, [])
```

Comment:
If challenge will always be a key of result please use `[]`. Otherwise, the current code will raise `TypeError` - please wrap it with specific exception